web/src/components/Feedback.tsx

```diff
@@ -141,10 +156,12 @@ const Feedback = (props: FeedbackProps): ReactElement => {
         <ErrorSendingStatus role='alert'>{t('failedSendingFeedback')}</ErrorSendingStatus>
       )}
 
-      <NoteContainer showContainer={sendFeedbackDisabled}>
-        <Icon src={NoteIcon} />
-        <NoteText>{t('note')}</NoteText>
-      </NoteContainer>
+      {!isSearchFeedback && (
```

Comment:
Instead of hiding this note you can just adjust the message for `isSearchFeedback` that shows up when the form field for missing search term is empty (deleted by user)

"Bitte geben sie den Suchbegriff ein für den sie kein Ergebnis finden konnten"